web/app/themes/clarity/inc/admin/prior-party/prior-party-banner-admin.php

```diff
@@ -324,7 +458,7 @@ public function menu(): void
             8
         );
 
-        add_action("load-$hook", [$this, 'pageLoad']);
+        // add_action("load-$hook", [$this, 'pageLoad']);
```

Comment:
@EarthlingDavey is this supposed to be commented out?

Reply:
Thanks, this is to remind me to run through my decisions with Damien tomorrow :)